sale_timesheet_line_exclude/security/exclude_from_sale_order.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright ACSONE SA/NV (https://acsone.eu)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="group_exclude_from_sale_order" model="res.groups">
+        <field name="name">Exclude timesheets from sale order</field>
+        <field name="category_id" ref="base.module_category_hidden" />
+        <field
+            name="comment"
+        >Gives access to the Non-Billable field on timesheets.</field>
```

Comment:
Maybe the role should also be added automatically to the Admin account?
`<field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>`